### ISSUE TITLE
Persistent caching 

### ISSUE THREAD

**Issue by pakammkrishna** [nicolasazrak/caddy-cache#19]
Caddy is losing caching information after server restart, then it takes time for downloading files for the first time. Is it possible to make caching persistent

**Comment by nicolasazrak** [comment]
Yes, this is definitely something that should be made. It's not a that simple because currently the information of cached responses is kept in memory, only saving the payloads in disk. I don't have much time right now but this is one of the next features I was planning.

**Comment by pataquets** [comment]
Also, a `default_max_age = forever` setting would be desirable, too.
